docs/concepts/command-service.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Command Service
+headline: Command Service
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Command Service</h2> 
+
+Commands service concept to be added.
```

Comment:
Have such TODO notes in a special style. It should be clearly visible that it's not a regular text.

Reply:
LGTM 
Agreed on using <p class="coming-soon">Coming soon</p> formatting style with web designer.